packages/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <div
+    :class="{'selection-dropdown-menu': appearance === 'selectionMenu'}"
+    class="k-dropdown k-dropdown-menu"
+  >
+    <KToggle v-slot="{toggle, isToggled}">
+      <KPop
```

Comment:
I think we should pass a `target` here so that for the dropdown, the popover is bound to the element instead of the body. Maybe `.k-dropdown-menu`

Reply:
Adding this causes the page to shift upward every time you click to open the menu. I remember seeing this on `KSelect` but can't remember how we fixed it. Need to investigate further

Reply:
Don’t forget to back this out

Reply:
Backed out